common/make/internal/executable_building_emscripten.mk

```diff
@@ -119,10 +119,10 @@ else ifeq ($(CONFIG),Debug)
 #
 # Explanation:
 # O0: Disable optimizations.
-# g4: Preserve maximum debug information, including source maps.
+# g3: Preserve maximum debug information, including DWARF data.
 EMSCRIPTEN_COMMON_FLAGS += \
   -O0 \
-  -g4 \
+  -g3 \
```

Comment:
I'd probably recommend to just use `-g` since that's the default equivalent to `-g3` and doesn't require looking up particular level in docs for the next person.

Reply:
Thanks for chiming in. Fair point, done.